test/scrabble_test.rb

```diff
@@ -1,13 +1,48 @@
 gem 'minitest'
-require_relative '../lib/scrabble'
+require './lib/scrabble'
 require 'minitest/autorun'
 require 'minitest/pride'
 require 'pry'
 
 class ScrabbleTest < Minitest::Test
+  def test_it_exists
+    scrabble = Scrabble.new
+
+    assert_instance_of Scrabble, scrabble
+  end
+
   def test_it_can_score_a_single_letter
-    skip
-    assert_equal 1, Scrabble.new.score("a")
-    assert_equal 4, Scrabble.new.score("f")
+    scrabble = Scrabble.new
+
+    assert_equal 1, scrabble.score("a")
+    assert_equal 4, scrabble.score("f")
+  end
+
+  def test_it_can_score_a_word
+    scrabble = Scrabble.new
+
+    assert_equal 8, scrabble.score("hello")
+  end
+
+  def test_multiplier_can_score_word
+    scrabble = Scrabble.new
+
+    assert_equal 9, scrabble.score_with_multipliers("hello", [1,2,1,1,1])
   end
+
+  def test_multiplier_can_score_word_with_multiplier_and_double_letter
+    scrabble = Scrabble.new
+
+    assert_equal 18, scrabble.score_with_multipliers("hello", [1,2,1,1,1], 2)
+  end
+
+  def test_it_can_score_with_bonus
+    scrabble = Scrabble.new
+
+    assert_equal 44, scrabble.score_with_multipliers("balloon", [1,1,3,1,1,2,1], 2)
+  end
```

Comment:
I like that you're testing a separate use case for for multipliers. I would have liked to see more of this for your other methods as well.